src/OpenFHE.jl

```diff
@@ -104,6 +114,9 @@ export SecretKeyDist, GAUSSIAN, UNIFORM_TERNARY, SPARSE_TERNARY
 export SecurityLevel, HEStd_128_classic, HEStd_192_classic, HEStd_256_classic,
        HEStd_128_quantum, HEStd_192_quantum, HEStd_256_quantum, HEStd_NotSet
 
+# native_int
+export get_native_int
+
```

Comment:
```suggestion
```
This is inconsistent. IMHO we should export both `get_native_int` and `set_native_int!` or neither. Given that it's a function likely used in special circumstances, I'd be OK either way.